src/libgriddyn/src/griddyn_run.cpp

```diff
@@ -0,0 +1,58 @@
+#include "griddyn_interface.hpp"
+
+void griddyn_sim_set_run_flags(griddyn_sim* ctx, griddyn_flag_t flags)
+{
+  griddyn::gridDynSimulation* sim_ptr = impl::sim_cast(ctx);
+  {
+    // TODO: There has got to be a better way of doing this
```

Comment:
I believe there is a set flags function that uses an index, not sure that is public or not